File: src/main/kotlin/aris/thesis/theratricalplaysapi/repositories/ChangeLogRepository.java
package aris.thesis.theratricalplaysapi.repositories;

import aris.thesis.theratricalplaysapi.entities.ChangeLog;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;

public interface ChangeLogRepository extends JpaRepository<ChangeLog, Integer>, JpaSpecificationExecutor<ChangeLog> {

}

File: src/main/kotlin/aris/thesis/theratricalplaysapi/repositories/ChangeLogRepository.kt
package aris.thesis.theratricalplaysapi.repositories

import aris.thesis.theratricalplaysapi.entities.ChangeLog
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor

interface ChangeLogRepository : JpaRepository<ChangeLog?, Int?>, JpaSpecificationExecutor<ChangeLog?>